### ISSUE TITLE
Timeline: option to disable autoplay

### ISSUE THREAD

**Issue by vaneenige** [motiondivision/motionone#22]
For manual control with the `.duration` property and progress `0...1` it's now required to call `.pause()` directly as timelines start automatically.

Would it be nice to have a `{ autoplay: false }` option on the timeline? :+1:

**Comment by JamesManningR** [comment]
Bumping this as I think this would be useful

**Comment by mattgperry** [comment]
This will be published in the next version. This is currently blocked by https://github.com/motiondivision/motionone/pull/240
